sprocs/variants_insert.sql

```diff
@@ -95,7 +95,7 @@ BEGIN
             ci.id = real_course_instance_id
             AND q.id = real_question_id;
 
-        IF course_id IS NULL THEN RAISE EXCEPTION 'inconsistent course for question_id and course_instance_id'; END IF;
+        -- IF course_id IS NULL THEN RAISE EXCEPTION 'inconsistent course for question_id and course_instance_id'; END IF;
```

Comment:
👀

Reply:
:joy: this is on my todo list to fix, don't worry.

Reply:
I need to merge with SQL changes from other PRs before I can fix this properly, it will be easier to do once the others are all in master.